tests/relational/conftest.py

```diff
@@ -147,7 +161,7 @@ def _setup_nba(
         referred_columns=["id"],
     )
 
-    return rel_data, states, cities, teams
+    yield rel_data, states, cities, teams
```

Comment:
I'm curious about using `yield` here instead of `return`. This extra `yield` is why you need to wrap it in `next()`, right?
Maybe that's necessary, e.g. if you need laziness of this, but I couldn't quite figure it out.

Reply:
I think this is left over from some earlier machete work—I've changed it back to a regular `return` and it works fine, thanks for the catch!